docs/ExamplesContents.md

```diff
@@ -0,0 +1,63 @@
+# Terraform Provider Documentation: Examples
+This file lists all the examples in each category.
+
+## Clusters
```

Comment:
There's really only a single example in Clusters, and that's at (https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb. The files in that directory are all supporting that one example.